RecoPPS/ProtonReconstruction/plugins/CTPPSProtonProducer.cc

```diff
@@ -108,6 +119,8 @@ CTPPSProtonProducer::CTPPSProtonProducer(const edm::ParameterSet &iConfig)
                  iConfig.getParameter<std::string>("multiRPAlgorithm"),
                  verbosity_),
       opticsValid_(false),
+      lhcInfoPerLSToken_(esConsumes<LHCInfoPerLS, LHCInfoPerLSRcd>(edm::ESInputTag("", lhcInfoPerLSLabel_))),
```

Comment:
Note that these could be simplified to
```suggestion
      lhcInfoPerLSToken_(esConsumes(edm::ESInputTag("", lhcInfoPerLSLabel_))),
```
(but keeping the explicit product and record types is also fine)